runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -578,6 +581,30 @@ public void onItemContainerChanged(ItemContainerChanged itemContainerChanged)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		if (!config.showFreezes())
```

Comment:
This seems to be reversed? Should be `if (config.showFreezes())`

Also, would be better as a guard clause:

```
if (!config.showFreezes())
{
    return;
}
```